pgcli/main.py

```diff
@@ -372,9 +375,17 @@ def run_cli(self):
 
         except EOFError:
             print ('Goodbye!')
-        finally:  # Reset the less opts back to original.
-            logger.debug('Restoring env var LESS to %r.', original_less_opts)
-            os.environ['LESS'] = original_less_opts
+        finally:
+            # Don't persist pgcli less defaults when session is terminated
+            if less_opts_adjusted:
+                os.environ['LESS'] = ''
```

Comment:
It seems like you're clearing the LESS env var, but if the user had customized their LESS env var, we want to restore it when we exit not overwrite it with empty string.